changelog.txt

```diff
@@ -1,3 +1,12 @@
+Version 3.3.0, April 1, 2020
+
+Implementation details:
+ * Removed `@DefaultInUncheckedCodeFor` and
+   `@DefaultQualifierInHierarchyInUncheckedCode`.  The conservative default
```

Comment:
Type system designer can override it via `org.checkerframework.framework.util.defaults.QualifierDefaults#addUncheckedCodeDefault`.  I think we want to keep that option, but not document it in the manual.